### PR TITLE
fix: typos in documentation files

### PR DESCRIPTION
**Pull request by leopardracer**
This pull request contains changes to improve clarity, correctness and structure.

**Description correction:**
Corrected `usng` to `using`

Please review the changes and let me know if any additional changes are needed.

### CODE REVIEW

**Comment by leopardracer** [comment]
@howardwu Hi! Apologies for the inconvenience. I just wanted to check if there’s anything I can improve in my pull request. Please feel free to ask any questions or let me know if any changes are needed to meet the project's merge requirements. I’m happy to make adjustments!

File: src/main/java/algebra/fft/README.md [hunk 1]
```diff
@@ -38,4 +38,4 @@ JavaPairRDD<Long, FieldT> radix2CosetInverseFFT(final JavaPairRDD<Long, FieldT>
 [CLRS] [_Introduction to Algorithms_](https://mitpress.mit.edu/books/introduction-algorithms), Thomas H. Cormen, Charles E. Leiserson, Ronald L. Rivest, and Clifford Stein, 2009
 
 [S11] [_Schönhage-Strassen Algorithm with MapReduce
-for Multiplying Terabit Integers_](https://people.apache.org/~szetszwo/ssmr20110429.pdf), Tsz-Wo Sze, 2011
\ No newline at end of file
+for Multiplying Terabit Integers_](https://people.apache.org/~szetszwo/ssmr20110429.pdf), Tsz-Wo Sze, 2011
```
Comment (howardwu) [inline src/main/java/algebra/fft/README.md:41]: Is this change necessary?

Comment (leopardracer) [reply src/main/java/algebra/fft/README.md:41]: No, I didn't change anything here.

Comment (howardwu) [reply src/main/java/algebra/fft/README.md:41]: Ok, then please revert this diff.

Comment (leopardracer) [reply src/main/java/algebra/fft/README.md:41]: Can I create new PR and change it?

Comment (howardwu) [reply src/main/java/algebra/fft/README.md:41]: Yes

Comment (leopardracer) [reply src/main/java/algebra/fft/README.md:41]: I’m not sure how to fix it, sorry. I created a new PR and there is the same error.
